frontend/components/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "vue-strap",
+  "version": "2.14.1",
+  "description": "Bootstrap components built with Vue.js",
+  "keywords": [
+    "bootstrap",
+    "vue-bootstrap",
+    "vue-component",
+    "vue",
+    "markbind"
+  ],
+  "homepage": "https://markbind.org",
```

Comment:
should this be https://markbind.org/userGuide/usingComponents.html?

Reply:
hmm might be better to just use `markbind.org` in case we shift `usingComponents` next time. it contains some irrelevant things like the page/sitenav too

Reply:
Ok.